src/app/lib/business/dad_jokes/dad_joke.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:app/access/dad_jokes/data/dad_joke_content_data.dart';
+import 'package:equatable/equatable.dart';
+
+/// Represents a dad joke.
+final class DadJoke extends Equatable {
+  final String id;
+  final String title;
+  final String selfText;
```

Comment:
What does selfText means?

Reply:
I think it's the joke itself, used the same name as `UnoApplicationTemplate`, I can probably make it clearer.